### PR TITLE
range frame was not showing

### PR DESCRIPTION
**Pull request by beldense**

### CODE REVIEW

**Comment by beldense** [comment]
do check the range frames that are updated for ulduar aswell, seems like to me some are missing these lines aswell

File: DBM-Naxx/MilitaryQuarter/Horsemen.lua [hunk 1]
```diff
@@ -50,6 +51,7 @@ function mod:OnCombatStart(delay)
 	NextBaronMark:Start()
 	NextThaneMark:Start()
 	
+	self:Schedule(225, DBM.RangeCheck.Show, DBM.RangeCheck, 12)
```
Comment (andrew6180) [inline DBM-Naxx/MilitaryQuarter/Horsemen.lua:54]: This just schedules it to show in 255 seconds. I don't think this is whats needed

File: DBM-Naxx/MilitaryQuarter/Horsemen.lua [hunk 2]
```diff
@@ -32,6 +32,7 @@ local NextThaneMark			= mod:NewNextTimer(15, THANE_MARK)
 local specWarnMarkOnPlayer	= mod:NewSpecialWarning("SpecialWarningMarkOnPlayer", nil, false, true)
 
 mod:AddBoolOption("HealthFrame", true)
+mod:AddBoolOption("RangeFrame")
```
Comment (andrew6180) [inline DBM-Naxx/MilitaryQuarter/Horsemen.lua:35]: This should fix the range check

**Review by andrew6180 — COMMENTED** [review]
fixed this in https://github.com/ajseward/DBM-Frostmourne/commit/4729b9cc06f856b50ad2bfc4bc94d08e1c8bfa9a thanks for pointing it out!

**Comment by beldense** [comment]
mod:AddBoolOption("RangeFrame", mod:IsRanged())
As you need a tank or dps to take some of the hits, I don't think isRanged is the solution here...
For Sir Zeliek boss

**Pull request closed without merging**